wpiutil/CMakeLists.txt

```diff
@@ -171,7 +175,7 @@ set_property(TARGET wpiutil PROPERTY FOLDER "libraries")
 
 target_compile_features(wpiutil PUBLIC cxx_std_17)
 if (MSVC)
-    target_compile_options(wpiutil PUBLIC /permissive- /Zc:throwingNew)
+    target_compile_options(wpiutil PUBLIC /permissive- /Zc:throwingNew /MP)
```

Comment:
Do we have to set MP for other libs too?

Reply:
Nope. Its transitive because of the public target option.